apps/fusion/project/ws/app/src/lib/routes/events/components/events-card/events-card.component.scss

```diff
@@ -67,6 +67,10 @@
     padding: 8px;
 }
 
+.avatars {
```

Comment:
**inline-flex** This class name you can use, it is already available. You don't have to create new styles